web/src/unread_ops.js

```diff
@@ -441,6 +444,7 @@ function process_scrolled_to_bottom() {
         return;
     }
 
+    assert(message_lists.current !== undefined);
```

Comment:
`narrow_state.is_message_feed_visible` can be changed to `message_lists.current !== undefined` to avoid this assert.

Reply:
Yeah, changed!